templates/project/Cargo.toml.liquid

```diff
@@ -28,15 +28,11 @@ features = [
 ]
 
 [dependencies]
-async-channel = "2"
-embedded-hal = { version = "~0.2", features = ["unproven"]}
-embedded-svc = "~0.28"
-futures-lite = "2"
 log = "0.4"
 
 [build-dependencies]
 cargo_metadata = "0.19"
-embuild = "0.32"
```

Comment:
This however was different from what was in `Micro-RDK's` top-level `Cargo.toml` so I have updated it.